src/diffusers/pipelines/flux/pipeline_flux.py

```diff
@@ -670,19 +827,27 @@ def __call__(
         lora_scale = (
             self.joint_attention_kwargs.get("scale", None) if self.joint_attention_kwargs is not None else None
         )
+        do_true_cfg = true_cfg > 1 or negative_prompt is not None
         (
             prompt_embeds,
             pooled_prompt_embeds,
             text_ids,
+            negative_prompt_embeds,
+            negative_pooled_prompt_embeds,
```

Comment:
can we actually make a `encode_prompt_cfg`? just so it's easier to know which part is from the original pipeline

Reply:
also we would not have to remove these `#Copied from`